src/XMLimport.cpp

```diff
@@ -1496,7 +1496,7 @@ int XMLimport::readScriptGroup(TScript* pParent)
             } else if (name() == "script") {
                 QString tempScript = readScriptElement();
                 if (!pT->setScript(tempScript)) {
-                    qDebug().nospace() << "XMLimport::readScriptGroup(...): ERROR: can not compile script's lua code for: " << pT->getName();
+                    qDebug().nospace() << "XMLimport::readScriptGroup(...): ERROR: can not compile script's lua code for: " << pT->getName() << "; error: " << pT->getError();
```

Comment:
Let's tidy it up a bit more:
* use `QDebug::noquote()` which means only quotation marks we provide are used (instead of wrapping each separate string)
* avoid multiple `:` near to each other - for `QDebug/QWarning` type messages I now tend to separate the word classifying things (in this case "ERROR") from the nature of the information that follows it just with a ` - `
* Since we are specifying a complete message sentence here we can provide the terminal punctuation - i.e. a '.'
```suggestion
                    qDebug().nospace().noquote() << "XMLimport::readScriptGroup(...) ERROR - can not compile script's lua code for \"" << pT->getName() << "\"; reason: " << pT->getError() << ".";
```